modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2849,8 +2849,19 @@ public CodegenParameter fromParameter(Parameter parameter, Set<String> imports)
             codegenParameter.vendorExtensions.putAll(parameter.getExtensions());
         }
 
-        if (parameter.getSchema() != null) {
-            Schema parameterSchema = ModelUtils.unaliasSchema(this.openAPI, parameter.getSchema());
+        Schema s;
+        if(parameter.getSchema() != null) {
+            s = parameter.getSchema();
+        } else if (parameter.getContent() != null) {
+            Content content = parameter.getContent();
+            MediaType mediaType = content.values().iterator().next();
+            s = mediaType.getSchema();
+        } else {
+            s = null;
```

Comment:
What about logging an error so that the user is informed of something wrong?

Reply:
Incase of null, there is already logging at https://github.com/OpenAPITools/openapi-generator/blob/b4fcf6f3d60f6ac45854cd109ff2581778541b72/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L3004 about the error hence I did not add any extra logging.